docs/translating.md

```diff
@@ -84,12 +85,85 @@ The second step is to upload the translatable strings to Transifex using
 The final step is to get the translated files from Transifex using
 `tx pull -a`.
 
+### Marking Strings for Translation
+When you run `python manage.py makemessages`, Zulip parses all the code
+for strings which are marked for translation and dumps these strings in
+a translation resource which is uploaded to the Transifex. Zulip provides
+syntax for marking strings in both Javascript files and Handlebar
```

Comment:
What is a "translation resource"? Could you link to or include a quick explanation/definition?